### ISSUE TITLE
Get "old" version of sunrise-sunset function from OEX Globe comp

### ISSUE THREAD

**Issue by MargaretSiple-NOAA** [afsc-gap-products/GAPsurvey#5]
Need to retrieve "old" version of sunrise-sunset function, update, and make it available here. Currently `get.sunrise.sunset()` uses the console instead of `tcltk` and doesn't have a great format for lat and long.

**Comment by EmilyMarkowitz-NOAA** [comment]
Note: 
The sunrise/sunset function you sent to me is now in the R package for you to edit when you are ready. I made some minor edits to the function but feel free to amend as needed: 

1) changed the name of the function to "get_sunrise_sunset" to match the naming style with the other functions
2) specified function variables within function calls (e.g., `crds` in `maptools::sunriset(
    crds = sp::SpatialPoints(...`)
3) added an example and notes to the documentation
4) maybe something else

**Comment by MargaretSiple-NOAA** [comment]
Awesome, thank you Em! Those all sound good to me. When I get my hands on the version that uses `tcltk`, I'll update it.

**Comment by EmilyMarkowitz-NOAA** [comment]
Also - would [code like this found in the `TEDtoBTD()`](https://github.com/afsc-gap-products/GAPsurvey/blob/main/R/functions.R#L152) be helpful for integrating `tcltk`?
```
  if(!quiet){
    tcltk::tkmessageBox(title = "Message",
                        message = paste0("Your new ", filename,
                                         " .BTD and .BTH files are saved."),
                        icon = "info", type = "ok")
  }
```

**Comment by sean-rohan-NOAA** [comment]
@EmilyMarkowitz-NOAA @MargaretSiple-NOAA I think this is closed by #10 that addressed #9?
